### ISSUE TITLE
Shortage of training and testing data.

### ISSUE THREAD

**Issue by Raman1121** [Raman1121/Notes_classifier#2]
Right now I have 300 and 1130 images of classes "Notes" and "Not-Notes" in the training set and 98 and 600 images of "Notes" and "Not-Notes"  in the test set. This is causing overfitting.
